packages/core/src/utils/CoreHelperUtil.ts

```diff
@@ -357,5 +357,36 @@ export const CoreHelperUtil = {
       address,
       type
     } as AccountTypeMap[N]
+  },
+
+  isCaipAddress(address?: unknown): address is CaipAddress {
+    if (typeof address !== 'string') {
+      return false
+    }
+
+    const sections = address.split(':')
+    const namespace = sections[0]
+
+    return sections.length === 3 && (namespace as string) in CommonConstants.CHAIN_NAME_MAP
+  },
+
+  parseCaipAddress(
```

Comment:
Can we import it from [here](https://github.com/reown-com/appkit/blob/e5a09bc20844b0e010a273eff12c3a31ca74c220/packages/common/src/utils/ParseUtil.ts#L15) ?